app/views/helpers/lists.php

```diff
@@ -144,15 +160,24 @@ public function displayRow(
             );
             echo format(__('created by {listAuthor}', true),
                         array('listAuthor' => $link));
+            
+            echo $this->Html->tag(
+                'span',
+                ' '.$this->Date->ago($createdDate),
+                array(
+                    'class' => 'date'
+                )
+            );
```

Comment:
It seems to me that no matter where we place the date it will feel kinda awkward. 

I wouldn't spend too much time on this right now, since there are plans to redesign of the UI to make it responsive, and we'll be reviewing every elements in every pages. When the time comes we'll rework completely the display of the lists.

So for now feel free to position the date where it feels best for you, even if it's not perfect. I'll make your changes available on dev.tatoeba.org so that the community can try out and see that's good enough for them. If not, I'll take care myself of re-adjusting the position of the date.